chef/cookbooks/neutron/recipes/post_install_conf.rb

```diff
@@ -84,7 +84,11 @@ def mask_to_bits(mask)
     end
   when 'gre'
     fixed_network_type = "--provider:network_type gre --provider:segmentation_id 1"
-    floating_network_type = "--provider:network_type gre --provider:segmentation_id 2"
+    if node[:network][:networks][:nova_floating][:use_vlan]
```

Comment:
maybe this should be moved after line 78? We should take care of the floating network just once not for every type (gre, vxlan, etc) so we can remove some lines like 81-83 etc